pkg/types/interfacetests/chain_reader_interface_tests.go

```diff
@@ -70,16 +70,25 @@ func runChainReaderGetLatestValueInterfaceTests[T TestingT[T]](t T, tester Chain
 				tester.SetTestStructLatestValue(t, &secondItem)
 
 				cr := tester.GetChainReader(t)
-				require.NoError(t, cr.Bind(ctx, tester.GetBindings(t)))
+				bindings := tester.GetBindings(t)
+
+				var bound types.BoundContract
+				for idx := range bindings {
+					if bindings[idx].Name == AnyContractName {
+						bound = bindings[idx]
+					}
+				}
+
+				require.NoError(t, cr.Bind(ctx, bindings))
```

Comment:
This code is repeated a lot, I am not sure why you don't just Bind everything like it used to do. Alternatively there should be a GetBinding() method that returns a specific bound contract

Reply:
I need to get a specific binding, but yes it is repeated a lot. I had thought about adding a new testing method to get specific bindings.